README.md

```diff
@@ -573,6 +573,49 @@ Don't worry about `adbkey` or `adbkey.pub` under `/.android`, not required.
 
   > Unfortunately it is not possible to pass through a USB device (or a serial port) to a container.
 
+## Firebase Test Lab 
+You can also run your UI tests on Firebase Test Lab using emulators and physical devices, to do so first you need to
+create a service account with an “editor” role in the [Google Cloud Platform](https://console.cloud.google.com/projectselector2/iam-admin/serviceaccounts?pli=1&supportedpurview=project) 
+and enable [Google Cloud Testing API and Google Cloud Results API](https://console.developers.google.com/apis/library?pli=1). Later, in your Google Cloud Console go to “Credentials” > “Create Credentials” > “Create Service Account Key” fill the data and select key type as JSON.
+
+
+* Now let's pull and run the Docker image with Google Cloud SDK
+
+```bash
+# pull the image
+docker pull thyrlian/android-firebase-test-lab
+# run the container passing the service account Json (at least the first time) as external volume
+docker run -d -p 2222:22 -v $(pwd)/sdk:/opt/android-sdk -v /path/on/host/auth.json:/auth.json thyrlian/android-firebase-test-lab
+```
+* Log in and run your tests
+
+```bash
+# check the running process and get a interactive shell
+Docker ps
+Docker exec -it <container-id> /bin/bash
+
+# check Google Cloud is installed properly
+gcloud info --format="value(installation.sdk_root)"
+# authenticate and create any config that you want
+gcloud auth activate-service-account -q --key-file myservicefile.json
+
+# list available devices to run your tests
+gcloud firebase test android models list
+
+# run your tests (make sure you have previosly assemble your app and test apk's)
+APK="--app=debug.apk --test=androidTest.apk"
+TYPE="instrumentation"
+DEVICES="--device model=Pixel2,version=27,locale=en,orientation=portrait" 
```

Comment:
Just a question: I guess you can run on multiple devices together, that's why you name it as plural, is this correct? :)

Reply:
yes :) , I'll add info about splitting an Espresso test-suite into multiple shards and execute them in parallel.

Reply:
Cool, thanks!